docker-compose.yml

```diff
@@ -50,11 +69,14 @@ services:
         condition: service_healthy
       redis:
         condition: service_healthy
+      email:
+        condition: service_healthy
     restart: unless-stopped
 
 volumes:
   db_data:
     external: false
   redis_data:
     external: false
-
+  email_data:
```

Comment:
I wasn't able to test sending an email successfully, so I'm not sure if messages are simply written to the container logs, or if they're persisted in a volume.

I suspect the former, in which case we wouldn't need the volume.

Reply:
Mesages are written to the `std out`. Removing the volume.